Specs/Scene/WebMapTileServiceImageryProviderSpec.js

```diff
@@ -86,6 +86,57 @@ describe("Scene/WebMapTileServiceImageryProvider", function () {
     expect(parseInt(queryObject.tilerow, 10)).toEqual(tilerow);
   });
 
+  it("generates expected tile urls for subdomains", function () {
```

Comment:
This looks good. After adding the changes to use subdomains when `useKvp` is true in `requestIamge()` and Fili's suggested change to the `url.match` constructor test this passes.